samples/file-encryptor/enclave/mbedtls_src/keys.cpp

```diff
@@ -87,7 +87,7 @@ int ecall_dispatcher::generate_password_key(
         (const unsigned char*)password, // Password to use when generating key
         strlen((const char*)password),  // Length of password
         salt,                           // salt to use when generating key
-        sizeof(salt),                   // size of salt
+        strlen((const char*)salt),      // size of salt
```

Comment:
BUG: strlen is not correct either.  See line 252 which passes in an array of random bytes of length SALT_SIZE_IN_BYTES
```suggestion
        SALT_SIZE_IN_BYTES,
```

Reply:
Changed strlen to SALT_SIZE_IN_BYTES